index.html

```diff
@@ -207,5 +207,53 @@ <h1>About me</h1>
             </ul>
         </div>
     </div>
+    <form action="https://formspree.io/f/mdovrwgl" method="post">
+        <span>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</span>
+        <ul>
+          <li>
+            <input type="text" aria-label="name" name="user_name" maxlength="30" placeholder="Full name" required/>
+          </li>
+          <li>
+            <input type="email" aria-label="mail" name="user_email" placeholder="Email address" required/>
+          </li>
+          <li>
+            <textarea id="msg" aria-label="user_message" maxlength="500" placeholder="Write me something..." required></textarea>
+          </li>
+          <li class="button">
+            <button type="submit">Get in touch</button>
```

Comment:
- [ ] [Optional] 👍 Great job on implementing the default stage style for your button! However, I noticed that you haven't implemented the hover and active stage styles as per the design. It's important to include these styles to provide a consistent and interactive user experience.

Please consider updating your button styles to include the hover and active stage styles as indicated in the design. 🙏 By incorporating these styles, you will enhance the user experience and ensure that your button aligns with the design requirements. Keep up the good work! 👍

✨![image](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/55840999/e8cd24a6-9c68-454a-ad69-bbd936f64029)